docs/bokeh/source/docs/first_steps/first_steps_9.rst

```diff
@@ -192,7 +192,7 @@ widgets on the left to change the sine wave on the right:
     </div>
 
 Find the source code for this Bokeh server app in the
-`Bokeh repository on GitHub <https://github.com/bokeh/bokeh/blob/master/examples/app/sliders.py>`_.
+:bokeh-tree:`examples/server/app/sliders.py`.
```

Comment:
This needs some minor tweak to be grammatical, currently it will render as 
> Find the source code for this Bokeh server app in the `examples/server/app/sliders.py`.
So e.g "in the repository file ..." or "in the ... file in the Bokeh repository", etc. something along those lines.